simple/stats/db.py

```diff
@@ -397,6 +406,34 @@ def __init__(self, db_params: dict[str, str]) -> None:
     for statement in _INIT_STATEMENTS:
       self.cursor.execute(statement)
 
+  def _maybe_create_database(connector: Connector,
+                             db_params: dict[str, str]) -> None:
+    kwargs = {
+        param: db_params[param]
+        for param in _CLOUD_MY_SQL_INSTANCE_CONNECT_PARAMS
+    }
+    db_name = db_params[CLOUD_MY_SQL_DB]
+    logging.info(
+        "Connecting to Cloud MySQL instance '%s' to check existence of DB '%s'.",
+        db_params[CLOUD_MY_SQL_INSTANCE], db_name)
```

Comment:
nit: maybe use a tmp var in place of two map refs

Reply:
Done.